tests/blocks/block_0.cairo

```diff
@@ -1,32 +1,32 @@
-// THIS CODE IS GENERATED BY SCRIPT, DO NOT EDIT IT MANUALLY
```

Comment:
Can you please keep this header? Sometimes there might be a temptation to do adhoc changes :)